documentation/Components.md

```diff
@@ -0,0 +1,128 @@
+# Components
+
+This file outlines and provides a brief description of the various components that make up the SDK. 
```

Comment:
Can we add a line here saying that the page is under construction. I feel bad for readers who might see that the expanders and then only after clicking it will they notice it's not filled in.

Reply:
We will add it in a new PR.